conf/defaults.ini

```diff
@@ -1478,6 +1478,8 @@ public_key_retrieval_disabled = false
 # Force download of the public key for verifying plugin signature on startup. If disabled, the public key will be retrieved every 10 days.
 # Requires public_key_retrieval_disabled to be false to have any effect.
 public_key_retrieval_on_startup = false
+# Enter a comma-separated list of plugin identifiers to skip loading (including core plugins). These plugins will be hidden in the catalog.
+skip_plugins =
```

Comment:
Nit. Would perhaps suggest renaming it to  `disabled_plugins`. Something with "skip" that does feel weird to read for me without being able to pinpoint exactly why :smile: